cypress/package.json

```diff
@@ -6,13 +6,15 @@
     "node": "18"
   },
   "scripts": {
-    "copy-locked": "cp ./cypress/e2e/saucedemo.locked.ts ./cypress/e2e/saucedemo.cy.ts",
```

Comment:
Why reorder ?
It was using alphabetical order for clarity.

Reply:
I wanted to have an order how a user should run them. 

1. cypress
2. cypress-modified
3. cypress-ignored